src/main/java/hu/dungeonhunter/tools/Fight.java

```diff
@@ -22,11 +27,16 @@
 
     @Setter
     @Getter
-    private int healingPotionCounter = 0;
+    private int randomEnemy;
+
+    @Getter
+    private boolean runningAwayFromGoblinKingTest;
+
+    @Getter
+    private boolean goblinKingIsDeadTest;
```

Comment:
You don't need this. You can get this from monster.isDefeat().

Reply:
i removed